tests/unit/api/neonDB.js

```diff
@@ -123,4 +123,22 @@ describe('NeonDB', function () {
     privNet.should.equal(customEndpoint)
     done()
   })
+
```

Comment:
if this is to test directly with the network, it should be done as integration. Unit tests are fully mocked to speed up development.